apps/the_monkeys/src/app/settings/components/security/Danger.tsx

```diff
@@ -30,8 +26,7 @@ export const Danger = ({ data }: { data?: IUser }) => {
         title: 'Success',
         description: 'Your account has been deleted successfully.',
       });
-
-      router.replace('/');
+      window.location.href = '/feed'; // this will force window to reload and push to feed
```

Comment:
A better fix would be to invalidate the query that are associated to user session.
As I can see in the code, that might have been misssed.
That way we don't have to to a full page reload

Reply:
This is clearner solultion without refreshing the full page - queryClient.setQueryData(['auth'], null);